modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -3195,7 +3104,7 @@ bool InputMediaStream_FFMPEG::read(unsigned char** data, int* size, int* endOfFi
 
         if (pkt_.stream_index != video_stream_id_)
         {
-            _opencv_ffmpeg_av_packet_unref(&pkt_);
+            compat::avPacketUnref(&pkt_);
```

Comment:
Please keep original function identifier in names (of the latest modern FFmpeg version).:
- original: av_packet_unref
- good compatibility wrapper name: `_opencv_ffmpeg_av_packet_unref`
- avoid that: `compat::avPacketUnref`

Approach:
```
... define _opencv_ffmpeg_av_packet_unref ...

#define av_packet_unref   av_packet_unref_must_be_wrapped  // guard

... other code without av_packet_unref() ...
```

Reply:
I've tried to place all functions to `compatibility` namespace with pascal case naming (keeping the original naming).
But I've got your point and change it to what you suggest. Thank you.